src/aura/BGE_DonationSelector/BGE_DonationSelector.cmp

```diff
@@ -59,28 +59,28 @@
         </lightning:layoutItem>
 
         <!--START UNPAID PAYMENTS CARDS-->
-        <aura:iteration var="donation" items="{!v.unpaidPayments}">
+        <aura:iteration var="payment" items="{!v.unpaidPayments}">
             <lightning:layoutItem largeDeviceSize="4"
                                   mediumDeviceSize="6"
                                   size="12"
                                   padding="around-small">
-                <lightning:card class="{!'slds-card_boundary sameCardHeight' + if(equals(donation.Id, v.selectedDonation.Id), ' selectedDonationCard')}">
+                <lightning:card class="{!'slds-card_boundary sameCardHeight' + if(equals(payment.Id, v.selectedDonation.Id), ' selectedDonationCard')}">
 
                     <aura:set attribute="title">
                         <lightning:icon iconName="standard:currency" size="small" class="slds-m-right_small" />
-                        <lightning:formattedText value="{!donation.Name}"/>
+                        <lightning:formattedText value="{!payment.Name}"/>
                     </aura:set>
 
                     <aura:set attribute="footer">
-                        <aura:if isTrue="{!equals(donation.Id, v.selectedDonation.Id)}">
+                        <aura:if isTrue="{!equals(payment.Id, v.selectedDonation.Id)}">
                             <div class="selectedDonationBody">
                                 <lightning:icon iconName="utility:check" size="small" class="slds-p-right_small selectedCheckmark"
                                                 alternativeText="{!$Label.c.stgCheckboxTrue}" title="{!$Label.c.stgCheckboxTrue}"/>
                                 <lightning:formattedText value="{!$Label.c.bgeFormSelectedPayment}"/>
                             </div>
                             <aura:set attribute="else">
                                 <lightning:button
-                                        value="{!donation}"
```

Comment:
This renaming is a great idea for clarity!

Reply:
👍🏽